src/help/validations.csv

```diff
@@ -3,47 +3,47 @@ A/B/C,A1,"TAS components: The combination of all the elements that make up the T
 A,A2,BudgetAuthorityAvailableAmountTotal_CPE = BudgetAuthorityAppropriatedAmount_CPE + BudgetAuthorityUnobligatedBalanceBroughtForward_FYB + AdjustmentsToUnobligatedBalanceBroughtForward_CPE + OtherBudgetaryResourcesAmount_CPE,Fatal Error,Implemented,Calculation
 A,A3,OtherBudgetaryResourcesAmount_CPE = ContractAuthorityAmountTotal_CPE + BorrowingAuthorityAmountTotal_CPE + SpendingAuthorityfromOffsettingCollectionsAmountTotal_CPE,Fatal Error,Implemented,Calculation
 A,A4,StatusOfBudgetaryResourcesTotal_CPE= ObligationsIncurredTotalByTAS_CPE + UnobligatedBalance_CPE,Fatal Error,Implemented,Calculation
-A,A6,"BudgetAuthorityAvailableAmountTotal_CPE= CPE value for GTAS SF 133 line #1910, for the same reporting period",Fatal Error,Implemented,Calculation
-A,A7,"BudgetAuthorityUnobligatedBalanceBroughtForward_FYB= value for GTAS SF 133 line #1000, for the same reporting period",Fatal Error,Implemented,Calculation
-A,A8,"BudgetAuthorityAppropriatedAmount_CPE= CPE aggregate value for GTAS SF 133 line #1160 + #1180 + #1260 + #1280, for the same reporting period",Fatal Error,Implemented,Calculation
-A,A9,"ContractAuthorityAmountTotal_CPE= CPE aggregate value for GTAS SF 133 line #1540 + #1640, for the same reporting period",Fatal Error,Implemented,Calculation
-A,A10,BorrowingAuthorityAmountTotal_CPE= aggregate value for GTAS SF 133 line #1340 + #1440 for the same reporting period.,Fatal Error,Implemented,Calculation
-A,A11,SpendingAuthorityfromOffsettingCollectionsAmountTotal_CPE= aggregate value for GTAS SF 133 line #1750 + #1850 for the same reporting period.,Fatal Error,Implemented,Calculation
-A,A12,AdjustmentsToUnobligatedBalanceBroughtForward_CPE= aggregate value for GTAS SF 133 line #1010 through 1042 for the same reporting period.,Fatal Error,Implemented,Calculation
-A,A14,GrossOutlayAmountByTAS_CPE= value for GTAS SF 133 line #3020 for the same reporting period.,Fatal Error,Implemented,Calculation
-A,A15,UnobligatedBalance_CPE= value for GTAS SF 133 line #2490 for the same reporting period.,Fatal Error,Implemented,Calculation
+A,A6,"BudgetAuthorityAvailableAmountTotal_CPE= CPE value for GTAS SF 133 line #1910, for the same reporting period","Warning currently, Fatal Error in future release",Implemented,Calculation

```

Comment:
it looks like this is reverting earlier changes. These should have been changed to errors but it looks like this PR is reverting it back to "Warning currently, error in future release"